Remove SMS from all notifcations except Incident Submitted

tinfernn opened the pull request:
# Feature Addition

CLOSES #233 

## Summary

Removes SMS from all notifications except for Incident Submitted notifications.

## Rationale

UPEI HSE only wants SMS notifications for new incident submissions.

## Design Documentation

N/A

## Changes

Removed 'vonage' from via() and toVonage() from the following:
- CommentAdded.php
- IncidentReviewRequestNotification.php
- InvestigationReturnedNotification.php
- InvestigationSubmittedNotification.php
- RootCauseAnalysisSubmittedNotification.php
- SupervisorAssignedNotification

## Impact

Removes ability for the previously mentioned notifications to send SMS.

## Testing

Removing SMS did not affect any existing tests.

## Screenshots/Video

N/A

## Checklist

- [x] Code follows the project's coding standards

- [ ] Unit tests covering the new feature have been added

- [x] All existing tests pass

- [ ] The documentation has been updated to reflect the new feature

## Additional Notes

N/A

tinfernn committed:
Removed 'vonage' from via() and toVonage() for the following notifications:
-CommentAdded.php
-IncidentReviewRequestNotification.php
-InvestigationReturnedNotification.php
-InvestigationSubmittedNotification.php
-RootCauseAnalysisSubmittedNotification.php
-SupervisorAssignedNotification

tinfernn committed:
Removed unused vonage imports for the following:
-CommentAdded.php
-IncidentReviewRequestNotification.php
-InvestigationReturnedNotification.php
-RootCauseAnalysisSubmittedNotification.php
-SupervisorAssignedNotification

tinfernn committed:
Fixed styling issue for the following:
-CommentAdded.php

OsmiumCoder commented on app/Notifications/Comment/CommentAdded.php:
```diff
@@ -6,7 +6,6 @@
 use Illuminate\Notifications\Notification;
-use Illuminate\Notifications\Messages\VonageMessage;
 
 class CommentAdded extends Notification
```
can you update this class to use base notification

tinfernn committed:
Changed the following for CommentAdded.php:
-Uses BaseNotification instead of Notification
-Removed unnecessary comment in constructor
